gm4_tnt_landmines/data/tnt_landmines/functions/disarming.mcfunction

```diff
@@ -0,0 +1,22 @@
+execute if entity @e[type=armor_stand,tag=gm4_tnt_mine,distance=..0.1,nbt={ArmorItems:[{},{},{},{id:"minecraft:crafting_table",Count:1b}]}] run summon item ~ ~1 ~ {Motion:[0.0,0.4,0.0],Item:{id:":player_head",Count:1b,tag:{SkullOwner:{Id:cabe7f2a-8a06-40e3-a089-f442a1c93902,Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NTQ0NzQyODI1NTUsInByb2ZpbGVJZCI6Ijc1MTQ0NDgxOTFlNjQ1NDY4Yzk3MzlhNmUzOTU3YmViIiwicHJvZmlsZU5hbWUiOiJUaGFua3NNb2phbmciLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzY2ODBmZmEyMTE5MzAwZGU2NWQzNjE4ODVkNjE0YjNlYTM3Y2QwODhiODc5OTAzMjc0MzNkNGMxMWU1ZjNjNSJ9fX0="}]}},gm4_tnt_landmines:{item:1,block:"crafting_table"},display:{Name:"{\"text\":\"Landmine\"}",Lore:["{\"text\":\"Drop to use\"}","{\"text\":\"Disguised as a crafting table\"}"]}}}}
+execute if entity @e[type=armor_stand,tag=gm4_tnt_mine,distance=..0.1,nbt={ArmorItems:[{},{},{},{id:"minecraft:netherrack",Count:1b}]}] run summon item ~ ~1 ~ {Motion:[0.0,0.4,0.0],Item:{id:":player_head",Count:1b,tag:{SkullOwner:{Id:b3a65b75-cbb1-435e-bebf-53d119587934,Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NTQ0NzQzNDQyNzYsInByb2ZpbGVJZCI6IjVkMjRiYTBiMjg4YzQyOTM4YmExMGVjOTkwNjRkMjU5IiwicHJvZmlsZU5hbWUiOiIxbnYzbnQxdjN0NGwzbnQiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzIwYmVlZWE4OTQ1MzAxOGMzNTVmZGZjZTg5YTM1ZTEyOGUwOWQyMzRlNWY5MGU0YzE4ZWE3MDI5NDYxOTFmNjIifX19"}]}},gm4_tnt_landmines:{item:1,block:"netherrack"},display:{Name:"{\"text\":\"Landmine\"}",Lore:["{\"text\":\"Drop to use\"}","{\"text\":\"Disguised as netherrack\"}"]}}}}
+execute if entity @e[type=armor_stand,tag=gm4_tnt_mine,distance=..0.1,tag=gm4_mine_invisible] run summon item ~ ~1 ~ {Motion:[0.0,0.4,0.0],Item:{id:"minecraft:player_head",Count:1b,tag:{SkullOwner:{Id:6fb6cb4a-b13a-4610-ad26-6d54ba77360b,Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NTQ0NzQzODQ0NTUsInByb2ZpbGVJZCI6IjU3MGIwNWJhMjZmMzRhOGViZmRiODBlY2JjZDdlNjIwIiwicHJvZmlsZU5hbWUiOiJMb3JkU29ubnkiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzVlY2NhZmZlN2UwMWJjNjFiNjdmMDZkNGRmNjU2ZjQxZjdjMGJkODE3NzllNjFkMmEzYjk2YjE0MDVlYWQ1ODYifX19"}]}},gm4_tnt_landmines:{item:1,block:"none"},display:{Name:"{\"text\":\"Landmine\"}",Lore:["{\"text\":\"Drop to use\"}","{\"text\":\"Insivble so dont forget its there!\"}"]}}}}
```

Comment:
Branch to a different function to avoid these repeated `@e` checks

Reply:
would that relay be needed in this case as it is only called when someone is disarming anyway which will not be that common?

Reply:
It's not super necessary but it's good practice imo